jwst/regtest/test_nirspec_fs_spec2.py

```diff
@@ -86,6 +105,31 @@ def test_nirspec_fs_spec2(run_pipeline, fitsdiff_default_kwargs, suffix):
     assert diff.identical, diff.report()
 
 
+@pytest.mark.bigdata
+@pytest.mark.parametrize(
+    'output',
+    ['jw01309022001_04102_00004_nrs2_photom.fits',
```

Comment:
photom and cal are identical (or at least they should be). Don't need both.

Reply:
ok fixed